Test/ContentPipeline/IntermediateDeserializerTest.cs

```diff
@@ -404,13 +404,15 @@ public void ExternalReferences()
         {
             Deserialize<ExternalReferences>("17_ExternalReferences.xml", externalReferences =>
             {
+                var pathSeparator = Path.DirectorySeparatorChar.ToString();
+
                 Assert.NotNull(externalReferences.Texture);
-                Assert.IsTrue(externalReferences.Texture.Filename.EndsWith(@"\Xml\grass.tga"));
+                Assert.IsTrue(externalReferences.Texture.Filename.EndsWith(pathSeparator + "Xml" + pathSeparator + "grass.tga"));
```

Comment:
I understand how this works now.  The filename stored in the object created by the deserialization gets the platform-specific path separators.  In XNA of course it used Windows path separators, but other platforms get the forward slashes.  So we need the test to check for the correct platform path separators.